app.yaml

```diff
@@ -54,3 +53,6 @@ handlers:
 - url: .*
   script: shell.application
   secure: always
+
+env_variables:
+  GOOGLE_APPLICATION_CREDENTIALS: "client-secret.json"
```

Comment:
This is a serious security issue, you cannot keep it in the root directory like this, otherwise people can read it via live shell:

```
open('client-secret.json').read()
```

We need to find another place (or another way) to store and read credentials for accessing Google cloud services.

On the same note, I would suggest to quickly delete your service account that was used for the last deploy, before it is compromised.

Reply:
This is a serious issue. I have closed the service account for now.

Reply:
I have deployed my changes without using any secret credentials, thus using the default AppEngine service.